johnslist/dbtest/templates/dbtest/base.html

```diff
@@ -46,8 +46,13 @@
                     <!--------- Logged in user navbar ------------>
 
                     {% if user.is_authenticated %}
+
                     <ul class="nav navbar-nav navbar-right">
                         <li><a>Welcome {{user.username}}!</a></li>
+                        <a  class="navbar-brand" rel="home" href="{% url 'notifications' %}" title="Notifications">
```

Comment:
We talked about this couple weeks ago, in your next iteration, I think we should have a dropdown list for this. If user clicks on "read more" or something like that, then we take them to notification page. See Facebook notification as an example.